modules/installation-gcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,77 @@
+// This file is referenced in the following assembly:
+// installing/installing_gcp/installing-gcp-shared-vpc.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-shared-vpc-config_{context}"]
+= Sample customized install-config.yaml file for shared VPC installation
+There are several configuration parameters which are required to install {product-title} on GCP using a shared VPC. The following is a sample `install-config.yaml` file which demonstrates these fields.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must modify this file with the correct values for your environment and cluster.
+====
+
+[source,yaml]
```

Comment:
Not sure how much folks are using this nowadays, but seems like a candidate for a collapsible block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#collapsible-content

Reply:
I'm going to leave it as is so that it's consistent with the sample yaml blocks in the rest of the install book.